atef/config.py

```diff
@@ -1207,6 +1207,21 @@ async def _compare(self, data: Any) -> Result:
             identifier=self.identifier
         )
 
+    async def compare(self) -> Result:
+        """
+        Run the inherited compare routine, but if signal is an enum,
+        also flip the string field and retry
+        """
+        result = await super().compare()
+
+        if (result.severity != Severity.success
+                and getattr(self.signal, 'enum_strs', None)):
+            self.comparison.string = not self.comparison.string
+            await self.get_data_async()
+            result = await super().compare()
```

Comment:
Will this give us confusing results in the cases where both versions of the check fail?